src/ledger/utils.ts

```diff
@@ -132,4 +134,27 @@ export function expandDerivationPath(derivationPath: string) {
 	const accountPath = derivationPath.substring(0, derivationPath.length - 4)
 	const signPath = derivationPath.substring(derivationPath.length - 3)
 	return { accountPath: accountPath, signPath: signPath }
+}
+
+export async function getPathsAndAddresses(network: string): Promise<DerivedAddress[]> {
+  const BASE_PATHS = "m/44'/60'/0'/0/"
```

Comment:
not sure that hardcoding 60 which is generally used for ethereum blockchain is correct or not.